insights/ui/navigator.py

```diff
@@ -0,0 +1,9 @@
+from insights.ui.locators import insight_page_locator
```

Comment:
change the name of the module everywhere to insights_page_locators instead of insight_page_locator.